pay-api/flags.json

```diff
@@ -2,6 +2,9 @@
     "flagValues": {
         "string-flag": "a string value",
         "bool-flag": true,
-        "integer-flag": 10
+        "integer-flag": 10,
+        "NEW_REFUNDED_STATUSES": true,
```

Comment:
in PROD (will need to setup):
NEW_REFUNDED_STATUSES = false 
CSO_BCOL_FIX = false
BAD_CSO_SERVICE_FEE = true